### PR TITLE
Add a "Getting started" banner to main command

### PR DESCRIPTION
**Pull request by kke**
Replaces #2250

Also adds a hint about logging in and reorders the command list (which makes the diff hard to review)

Banner:
image

Login hint:
image

### CODE REVIEW

File: cli/lib/kontena/main_command.rb [hunk 1]
```diff
@@ -9,25 +9,31 @@ class Kontena::MainCommand < Kontena::Command
...
   subcommand "etcd", "Etcd specific commands", load_subcommand('etcd_command')
+  subcommand "certificate", "LE Certificate specific commands", load_subcommand('certificate_command')
+  subcommand "registry", "Registry specific commands", load_subcommand('registry_command')
   subcommand "external-registry", "External registry specific commands", load_subcommand('external_registry_command')
+  subcommand "volume", "Volume specific commands [EXPERIMENTAL]", load_subcommand('volume_command')
+  subcommand "app", "App specific commands", load_subcommand('app_command')
+  subcommand "plugin", "Plugin specific commands", load_subcommand('plugin_command')
   subcommand "whoami", "Shows current logged in user", load_subcommand('whoami_command')
-  subcommand "plugin", "Plugin related commands", load_subcommand('plugin_command')
+  subcommand "logout", "Logout from Kontena Masters or Kontena Cloud accounts", load_subcommand('logout_command')
```
Comment (SpComb) [inline cli/lib/kontena/main_command.rb]: IMO the deprecated `logout` command should either be removed, or preferrably just hidden from the `--help` output.

The `logout` description that shows up in the `--help` is misleading without any indication of it being a deprecated command that doesn't do anything.

Comment (kke) [inline cli/lib/kontena/main_command.rb]: True

**Comment by SpComb** [comment]
Weird merge conflict on the completer.

**Comment by kke** [comment]
Yeah the whole thing in HEAD is indented due to begin/rescue.

**Pull request merged**